ee/api/test/test_billing.py

```diff
@@ -397,3 +399,38 @@ def mock_implementation(url: str, headers: Any = None, params: Any = None) -> Ma
             },
         }
         assert self.organization.customer_id == "cus_123"
+
+    @patch("posthog.demo.matrix.manager.bulk_queue_graphile_worker_jobs")
+    @patch("posthog.demo.matrix.manager.copy_graphile_worker_jobs_between_teams")
+    @patch("ee.api.billing.requests.get")
+    def test_organization_usage_count(self, mock_request, *args):
+        self.organization.customer_id = None
+        self.organization.usage = None
+        self.organization.save()
+        # Create a demo project
+        self.organization_membership.level = OrganizationMembership.Level.ADMIN
+        self.organization_membership.save()
+        response = self.client.post("/api/projects/", {"name": "Test", "is_demo": True})
+        self.assertEqual(response.status_code, 201)
+        self.assertEqual(Team.objects.count(), 3)
+
+        mock_request.return_value.status_code = 200
+        mock_request.return_value.json.return_value = create_billing_response(
+            # Set usage to none so it is calculated from scratch
+            customer=create_billing_customer(has_active_subscription=False, usage=None)
+        )
+
+        assert not self.organization.usage
+        res = self.client.get("/api/billing-v2")
+        assert res.status_code == 200
+        self.organization.refresh_from_db()
+        assert self.organization.usage == {
+            "events": {
+                "limit": 10000,
+                "usage": 0,
```

Comment:
This test is passing, but doesn't actually do what I think it will. I create a demo project above, but in doing so the demo project doesn't actually have any events, so we can't confirm that the demo project is being excluded from the counts via this test since it has no events to be counted in general. 

I expected that doing this would actually put events in the demo project like we see in prod, but I don't think it does. I'm actually fine with it not doing that, since it would slow the test down considerably, but I'm not sure how to mock an event count for the team. I can figure it out when I am back next week but suggestions would be helpful :)